app/views/dashboard/_publishers.html.erb

```diff
@@ -23,3 +23,75 @@
     <% end %>
   <% end %>
 </table>
+<script>
```

Comment:
@gkostin1966 @sethaj @conorom General housekeeping question - are we okay with inline JS like this, or should this be stored and minified with the rest of the app's JS in the `assets` directory? Maybe the file would be called something like  `/app/assets/javascripts/dashboard_sort.js`?